Userland/Utilities/stty.cpp

```diff
@@ -589,34 +583,24 @@ int main(int argc, char** argv)
     ScopeGuard file_close_guard = [&] { close(terminal_fd); };
 
     termios initial_termios;
-    if (tcgetattr(terminal_fd, &initial_termios) < 0) {
-        perror("tcgetattr");
-        exit(1);
-    }
+    TRY(Core::System::tcgetattr(terminal_fd, &initial_termios));
```

Comment:
When using `ErrorOr` the out param is returned because the return value can mean a value or error.
That means you need to use the return value of `Core::System::tcgetattr`.
```c++
// construct it from
termios initial_termios{TRY(Core::System::tcgetattr(terminal_fd))};
// or assign it
termios initial_termios=TRY(Core::System::tcgetattr(terminal_fd));
```

Reply:
Thank you for the explanation.